### ISSUE TITLE
Missing whitespace around export statement with "check-module" argument

### ISSUE THREAD

**Issue by hdurix** [palantir/tslint#4014]
### Bug Report

- __TSLint version__: 5.10.0
- __TypeScript version__: 2.9.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
import MockedForm from './MockedForm';

export {MockedForm};
```

with `tslint.json` configuration:

```json
{
  "defaultSeverity": "warning",
  "extends": [
    "tslint:recommended"
  ],
  "linterOptions": {
    "exclude": [
      "node_modules/**"
    ]
  },
  "rules": {
    "quotemark": [true, "single"],
    "indent": [true, "spaces", 2],
    "interface-name": false,
    "object-literal-sort-keys": false,
    "whitespace": [true, "check-branch", "check-decl", "check-operator", "check-separator", "check-type", "check-module"]
  }
}
```

#### Actual behavior

No issue is detected by tslint.

#### Expected behavior

Add whitespace around export statements when missing. Because of rule "whitespace" with argument "check-module".

```ts
import MockedForm from './MockedForm';

export { MockedForm };
```
